spec/controllers/book_suggestions_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+
+describe BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    context 'When creating a book suggestion' do
+      let!(:book_suggestion) { create(:book_suggestion) }
+
+      subject do
+        post :create
+      end
+
+      it 'responds with the book suggestion json' do
+        expected = book_suggestion.to_json
+        expect(response.body.to_json) =~ JSON.parse(expected)
+      end
+
+      it 'responds with 201 status' do
+        expect(response).to have_http_status(:success)
```

Comment:
you can be a little bit more specific with `:created` (201) rather than `:success` (2XX)